Tools/TestWebKitAPI/Tests/WebKitCocoa/WKContentExtensionStore.mm

```diff
@@ -1294,7 +1294,10 @@ HTTPServer server({
     EXPECT_WK_STREQ([webView _test_waitForAlert], "original page loaded");
     [webView evaluateJavaScript:@"window.location = 'https://webkit.org/before_redirect'" completionHandler:nil];
     EXPECT_WK_STREQ([webView _test_waitForAlert], "loaded replacement successfully: https://apple.com/after_redirect");
-    // FIXME: <rdar://149306346> going back should arrive back at the original page.
+
+    auto *backItem = webView.get().backForwardList.backItem;
+    EXPECT_WK_STREQ(backItem.URL.absoluteString, [NSURL URLWithString:@"https://example.com/example"].absoluteString);
+
```

Comment:
Could you add a call to WKWebView.goBack then wait for another alert and verify the original page loads?